java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/db/DatabaseCommands.java

```diff
@@ -0,0 +1,160 @@
+                }
+            }
+        }
+
+        return client.showQuickPick(new ShowQuickPickParams(Bundle.MSG_SelectDriver(), false, items))
+                .thenApply(selectedItems -> {
+                    if (selectedItems == null) {
+                        return null;
+                    }
+                    if (!selectedItems.isEmpty()) {
+                        int i = ((Double) selectedItems.get(0).getUserData()).intValue();
+                        JDBCDriver driver = drivers[i];
+                        client.showInputBox(new ShowInputBoxParams(
+                                Bundle.MSG_EnterDbUrl(), "")).thenAccept((u) -> {
+                            if (u == null) {
+                                return;
+                            }
+                            client.showInputBox(new ShowInputBoxParams(
+                                    Bundle.MSG_EnterUsername(), "")).thenAccept((username) -> { //NOI18N
+                                if (username == null) {
+                                    return;
+                                }
+                                client.showInputBox(new ShowInputBoxParams(
+                                        Bundle.MSG_EnterPassword(), "", true)).thenAccept((password) -> { //NOI18N
+                                    if (password == null) {
+                                        return;
+                                    }
+                                    DatabaseConnection dbconn = DatabaseConnection.create(driver, u, username, null, password, true);
+                                    try {
+                                        ConnectionManager.getDefault().addConnection(dbconn);
+                                        List<String> schemas = getSchemas(dbconn);
+                                        System.out.println("schemas " + schemas);
+                                        if (schemas.isEmpty()) {
+                                            client.showMessage(new MessageParams(MessageType.Info, Bundle.MSG_ConnectionAdded()));
+                                        } else {
+                                            List<QuickPickItem> schemaItems = schemas.stream().map(schema -> new QuickPickItem(schema)).collect(Collectors.toList());
+                                            client.showQuickPick(new ShowQuickPickParams(Bundle.MSG_SelectSchema(), schemaItems)).thenAccept((s) -> {
+                                                if (s == null) {
+                                                    try {
+                                                        // Command was interrupted, we don't want the connection to be added
+                                                        ConnectionManager.getDefault().removeConnection(dbconn);
+                                                    } catch (DatabaseException ex) {
+                                                        System.out.println(ex.getMessage());
```

Comment:
This should either be a logger or a call to `org.openide.util.Exceptions#printStackTrace`.